### PR TITLE
[Csv] add tests for csv dataset script

### PR DESCRIPTION
**Pull request by patrickvonplaten**
Adds dummy data tests for csv.

### CODE REVIEW

File: datasets/csv/csv.py [hunk 1]
```diff
@@ -11,10 +10,11 @@
 @dataclass
 class CsvConfig(nlp.BuilderConfig):
     """BuilderConfig for CSV."""
+
     skip_rows: int = 0
     header_as_column_names: bool = True
     delimiter: str = ","
-    quote_char: str = "\""
+    quote_char: str = '"'
```
Comment (patrickvonplaten) [inline datasets/csv/csv.py:17]: @thomwolf - black did this automatically - is that ok?

File: datasets/csv/csv.py [hunk 2]
```diff
@@ -2,7 +2,6 @@
 
 from dataclasses import dataclass
 
-import pyarrow as pa
```
Comment (patrickvonplaten) [inline datasets/csv/csv.py:5]: Deleted this import since it doesn't seem to be in use @thomwolf

File: tests/test_dataset_common.py [hunk 3]
```diff
@@ -91,7 +92,16 @@ def check_load_dataset(self, dataset_name, configs, is_local=False):
                     is_local=is_local,
                 )
 
-                # build dataset from dummy data
+                if dataset_builder.__class__.__name__ == "Csv":
```
Comment (patrickvonplaten) [inline tests/test_dataset_common.py:95]: @thomwolf - a bit hacky here, but I think it's ok for now

**Comment by patrickvonplaten** [comment]
@thomwolf - can you check and merge if ok?

File: datasets/csv/csv.py (hunk 1 above)
Comment (thomwolf) [reply datasets/csv/csv.py:17]: Yes!

File: datasets/csv/csv.py (hunk 2 above)
Comment (thomwolf) [reply datasets/csv/csv.py:5]: Ok!

File: tests/test_dataset_common.py (hunk 3 above)
Comment (thomwolf) [reply tests/test_dataset_common.py:95]: Ok! I trust you with the tests :-)

**Pull request merged**